Dockerfile

```diff
@@ -1,3 +1,4 @@
+# Generated by https://smithery.ai. See: https://smithery.ai/docs/config#dockerfile
```

Comment:
I will just ask you to just remove this line, since it was not generated with Smithery.

Reply:
It was generated! I'm just making the PR.